src/WidgetApi.ts

```diff
@@ -432,8 +437,24 @@ export class WidgetApi extends EventEmitter {
                 content,
                 ...(stateKey !== undefined && { state_key: stateKey }),
                 ...(roomId !== undefined && { room_id: roomId }),
-                ...(futureTimeout !== undefined && { future_timeout: futureTimeout }),
-                ...(futureGroupId !== undefined && { future_group_id: futureGroupId }),
+                ...(delay !== undefined && { delay }),
+                ...(parentDelayId !== undefined && { parent_delay_id: parentDelayId }),
+            },
+        );
+    }
+
+    /**
+     * @deprecated This currently relies on an unstable MSC (MSC4157).
```

Comment:
is deprecated what we use here?
do we have an @unstable?

Reply:
Yes -- for whatever reason, other MSC-dependent methods in `src/WidgetApi.ts` are documented with `@deprecated` instead of `@unstable`, so I followed suit here.